### PR TITLE
add event 2016-06-11

### PR DESCRIPTION
**Pull request by khoeger**
Add event New Musical Instruments 2016 (6/11/2016)

### CODE REVIEW

**Review by anilmuppalla — APPROVED** [review]

**Review by sunnythere — APPROVED** [review]

File: _events/2016-06-11.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Scott Barton & Steven Kemper
+- Jessica Segal
+- Matthew Ostrowski
+
+# Workshops
+
+- Jay Allen Zimmerman, Roboticus
+
+# Details
+The [Monthly Music Hackathon]({{site.baseurl}}) series presents our [New Musical Instruments Hackathon]({{site.baseurl}}{{link _events/2016-06-11.md}} "New Musical Instruments Hackathon 2016"):  a day-long, non-competitive, creative event exploring new ways to make music, noise, & sound.  From musical robots to hyperinstruments to making wind instruments out of pipes, we’ve got it all. The event is open to hackers and aspiring hackers of all skill levels and disciplines, utilizing technology or not.
```
Comment (trevorknight) [inline _events/2016-06-11.md]: > [New Musical Instruments Hackathon]({{site.baseurl}}{{link _events/2016-06-11.md}}

It's a little weird for this page to link to itself. I'd vote for just removing both of these links.

File: _events/2016-06-11.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Our presenters include [Thomas Carroll](http://www.carrollclarinet.com/ "Thomas Carroll"), who builds clarinets and historical wind instruments; [Lori Napoleon (Merdian7, Antenes)](http://meridian7.net/ "Lori Napoleon"), who builds modular synths from old telephones switchboards; Scott Barton & Steven Kemper, who make musical robots; [Jessica Segal](http://www.jessicasegall.com/ "Jessica Segal"), who makes music from bees and old pianos; and Matthew Ostrowski, who makes music out of rotary phones.
+
+We’re also planning workshops on new instrument hardware, interactive dance & gestural music. Performance-wise, we’ll have an excerpt of [Jay Allen Zimmerman](http://www.musicbyjaz.com "Jay Allen Zimmerman")’s Roboticus, along with music from the EMMI bots.
```
Comment (trevorknight) [inline _events/2016-06-11.md]: In general, I don't think we need to write the link title/hover text.
I mean, just more effort for not much gained. I dunno; what do you think?

**Review by trevorknight — APPROVED** [review]

File: _events/2016-06-11.md (hunk 2 above)
Comment (anilmuppalla) [reply _events/2016-06-11.md]: I agree.

Comment (khoeger) [reply _events/2016-06-11.md]: Ok! Next time, I shall skip that.

**Comment by khoeger** [comment]
I realized that the link to the MMH site is not pointing to the about page... I've made another change to this branch that will fix this. Shall commit it.

File: _events/2016-06-11.md (hunk 1 above)
Comment (khoeger) [reply _events/2016-06-11.md]: Ah, yes! I'll remove the second link.

**Comment by khoeger** [comment]
I've put a new version up.

**Review by trevorknight — APPROVED** [review]

**Comment by trevorknight** [comment]
By the way, please check out this link on best practices for commit comments: https://github.com/blog/926-shiny-new-commit-styles

**Pull request merged**